tests/ga/test_update.py

```diff
@@ -2522,79 +2525,65 @@ def __init__(self, return_value=0, side_effect=None):
         self.wait = Mock(return_value=return_value, side_effect=side_effect)
 
 
-class ExtensionsGoalStateMock(object):
-    def __init__(self, identifier):
-        self.id = identifier
-
-
 class GoalStateMock(object):
-    def __init__(self, incarnation):
+    def __init__(self, incarnation, family, versions):
```

Comment:
Some of the logic in the production code was moved from WireProtocol to GoalState. Updating the mocks to match those changes. These mocks need to be rewritten.